lstchain/onsite.py

```diff
@@ -116,7 +116,7 @@ def find_pedestal_file(pro, pedestal_run=None, date=None, base_dir=DEFAULT_BASE_
 
     if pedestal_run is not None:
         # search a specific pedestal run
-        file_list = sorted(ped_dir.rglob(f'{pro}/drs4_pedestal.Run{pedestal_run:05d}.0000.h5'))
+        file_list = sorted(ped_dir.rglob(f'*/{pro}/drs4_pedestal.Run{pedestal_run:05d}.0000.h5'))
```

Comment:
is this really needed? You have already the full path where you expect the drs4 ped files.
```
base_dir / CAT_A_PIXEL_DIR / "drs4_baseline" / rglob over date directories / {pro}/drs4_pedestal.Run{pedestal_run:05d}.0000.h5'
```
I don't know if it makes any difference to add the wild card here unless you have a different directory structure in between.